app/popup/popup.js

```diff
@@ -196,6 +194,112 @@ function reloadPopup(tabID) {
       request.showLazyloadedImages();
     });
 
+    document.getElementById("parent-list").addEventListener("click", function (e) {
+      if (e.target && e.target.nodeName == "INPUT") {
+        jeclique(e.target.id);
+      }
+    });
+    
+    function jeclique(cookieName) {
+      var toggle = true;
+      toggle = document.getElementById(cookieName).checked;
+      if (toggle == false) {
+        browser.tabs.query({ active: true, lastFocusedWindow: true }).then(tabs => {
```

Comment:
The code is duplicated. The only change is `m_dict = this.getParamsDict(cookieName, false | true)`